pkg/server/commands/list_objects_test.go

```diff
@@ -45,15 +45,16 @@ func TestListObjectsDispatchCount(t *testing.T) {
 	}{
 		{
 			name: "test_direct_relation",
-			model: `model
-			schema 1.1
+			model: `
+model
+schema 1.1
 
-			type user
+type user
 
-			type folder
-				relations
-					define viewer: [user] 
-			`,
+type folder
+	relations
+		define viewer: [user] 
+	`,
```

Comment:
I liked it better the way it was before 
<img width="619" alt="image" src="https://github.com/openfga/openfga/assets/5374887/e526885e-d8e0-44d5-ac44-b36f44c2824b">